release-team/release-team-selection.md

```diff
@@ -0,0 +1,102 @@
+
+If there are more contributors interested in shadowing once hitting that upper bound, we welcome them to sit in on Release Team meetings in preparation for shadowing in a future release cycle.
+
+In Kubernetes 1.14, we begun handling shadow selection using a questionnaire, as opposed to the first-come, first-served GitHub issue approach that we had adopted during previous release cycles ([1.12](https://github.com/kubernetes/sig-release/issues/167), [1.13](https://github.com/kubernetes/sig-release/issues/280).
+
+Moving to a [questionnaire](#sample-questionnaire) / survey format will allow us to capture more of the information required to vet potential shadows and well as begin to generate metrics around some of the Release Team processes.
+
+Questionnaire creation will be facilitated by SIG Release Chairs or a delegated coordinator to ensure consistency across release cycles.
+
+Following submissions, the questionnaires will be reviewed by role leads of the incoming Release Team.
+
+Volunteers that meet the requirements of the respective role will be contacted by those leads via some convenient mechanism (Slack, video chat, etc.) to further discuss.
+
+After vetting the volunteers for their roles, role leads should make a final decision on selected shadows with the incoming Release Team Lead.
+
+
+### Sample Questionnaire
+
+Thank you for volunteering for the Kubernetes 1.14 Release Team! 
+
+This questionnaire is meant to learn a little bit more about you, your journey with Kubernetes, your understanding of the Release Team processes and procedures, as well as to determine if you're a good fit for the current Release Team. Please note that you are expected to be inexperienced in some areas, as a role shadow is a mentorship opportunity.
+
+Participation in the Release Team can include somewhat open-ended expectations. We never know what blocking/critical issue may come up at what point during the release cycle. On a rare occasion this may mean we each make some personal sacrifice to check in on a status or show up at a meeting to give or receive information. The team of leads and shadows must be flexible and pragmatic in addressing these. The Release Team Lead will endeavor to make sure any such inconveniences do not unfairly hit individuals or specific timezones.
+
+#### Logistics
+
+- What [Release Team role(s)](https://git.k8s.io/sig-release/release-team#kubernetes-release-team-roles) are you interested in shadowing? _[Select up to two (2)]_
+- The [Role Handbooks](https://git.k8s.io/sig-release/release-team/role-handbooks) defines the tasks and responsibilities of each role for this release cycle. Are you prepared to assist in fulfilling these duties? You must read the handbook relevant to the role you're interested in before answering.
```

Comment:
This is a rubber stamp question.  How do I know they've read the handbook?  Again, I won't hold up for this.

But consider questions that demonstrate a bit of comprehension, eg: 
- "how many hours a week do you think you'll need to fulfill these duties?"
- "which parts interest you most?"
- "which parts interest you least?"

Reply:
I'm inclined to hope that people are responding factually on the questionnaire, and by extension, have actually read the relevant handbooks. I generalized some of the questions so it can be used for all roles.

...just want to make sure this is more like getting a baseline of the volunteer and less like grading a test.

The latter two make for good "exit interview" questions. Maybe we can have a post-release survey in future?